README.md

```diff
@@ -63,13 +66,81 @@ builder
             javascript: {
                 enabled: true,
                 typescript: true,
+                react: true,
```

Comment:
er is toch geen react setting ?

Reply:
geen idee, zag dat ik  hierboven `tsx` gebruikte. Zal 'm weghalen ;)

Reply:
https://github.com/freshheads/webpack-config-builder/pull/50/commits/b950b89b23cf4447cc77b96ece498ac33ff35a53